test/cli.js

```diff
@@ -110,3 +115,12 @@ test('pkg-conf: cli takes precedence', function (t) {
 		t.end();
 	});
 });
+
+test('handles NODE_PATH', function (t) {
+	var nodePaths = 'node-paths/modules' + path.delimiter + 'node-paths/deep/nested';
```

Comment:
@sindresorhus do you mean `path.join('node-paths', 'modules')`? `path.delimiter` is used for building the environment variable itself, which comprises multiple paths.